.rubocop.yml

```diff
@@ -1,496 +1,12 @@
-inherit_from: .rubocop_todo.yml
-require:
-  - rubocop-rspec
-  - rubocop-rails
+require: standard
```

Comment:
Rubocop config shrinks to a 9-line file since standard now provides the bulk of rubocop configuration.